rhdm70-image-streams.yaml

```diff
@@ -13,7 +13,9 @@ items:
     annotations:
       openshift.io/display-name: Red Hat Decision Manager Central 7.0
       openshift.io/provider-display-name: Red Hat, Inc.
+      #openshift.io/image.insecureRepository: 'true' #Please Uncomment the insecureRepository if your repo is not secured 
```

Comment:
So I have put it into comment status. So this can facilitate a user to notify that this setting is required if he has to use an insecure repository. It seems to me that the insecure registry is quite common in developing mode.
Is it a valid case for a user to use an insecure repository in dev mode?

Reply:
I think it is more a doc issue than the imagestream itself, I think better we document the needed steps if the user wants to use a insecure registry.

Reply:
OK, I am fine with this.

Reply:
@spolti So I removed the line you mentioned. 
Could you do a review on the rest?